examples/titanic/assets/algo_random_forest/algo.py

```diff
@@ -130,7 +130,7 @@ def train(self, X, y, models, rank):
                                                oob_score=True,
                                                random_state=1,
                                                n_jobs=-1)
-        random_forest.fit(X, y)
+        random_forest.fit(X, y.values.ravel())
```

Comment:
was the error due to a specific version of sklearn ? Since I did not reproduce. If that's the case, could you fix the version of sklearn in the Dockerfile ? and maybe add a comment in the README that for these manual tests you need  the right version for the libraries

Reply:
It is a warning, not an error in my case:

```DataConversionWarning: A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples,), for example using ravel().```

while `y` is a Pandas dataframe of shape [712 x 1]

I have sklearn v0.2.4, though I am not convinced this is the problem of the version

Still, as it is surprising that you are not seeing it, I will fix the version. It won't harm

Reply:
It is a warning, not an error in my case:

DataConversionWarning: A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples,), for example using ravel().

while y is a Pandas dataframe of shape [712 x 1]

I have sklearn v0.2.4, though I am not convinced this is the problem of the version

Still, as it is surprising that you are not seeing it, I fixed the version in the dockerfile. It won't harm

ps. If I run this same algo.py in the debug mode I will see the same warning so it is not only related to the manual tests